monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy

```diff
@@ -0,0 +1,96 @@
+package com.novoda.monkey
+
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+public class MonkeyConfigurationPlugin implements Plugin<Project> {
+
+    private static final String EXTENSION_NAME = 'monkeyRunner'
```

Comment:
I think `MONKEY_RUNNER_EXTENSION` would get more readable, the same for the other constants, let's be more explicit